UnityProject/Assets/Scripts/Items/Weapons/Gun.cs

```diff
@@ -170,8 +169,18 @@ public class Gun : NetworkBehaviour, IPredictedCheckedInteractable<AimApply>, IC
 
 		private GunTrigger gunTrigger;
 
+		/// <summary>
+		/// If true, displays a message whenever a gun is shot
+		/// </summary>
+		[SyncVar(hook = nameof(SyncIsSuppressed)), Tooltip("If the gun displays a shooter message")]
 		public bool isSuppressed;
+
+		/// <summary>
+		/// Enables or disables the behaviour related to applying and removing suppressors from the gun
+		/// </summary>
+		[Tooltip("If suppressors can be applied or removed")]
 		public bool isSuppressible;
```

Comment:
will ``isSuppressible`` ever change? I think this could be a private serialized field if we don't need to get the value anywhere and we don't expect the value to change after the object has spawned.

Reply:
several weapons start suppresed (energy crossbow)

Reply:
I didn't understand. You need to get this variable elsewhere? If you want to read it on a child class of ``Gun`` you could either do a public getter or make it ``protected``.